### DISCUSSION TITLE
Install Git

### DISCUSSION THREAD

**Discussion by AnnaBlatt** [irgroup-classrooms/dis08-2021#12] [Q&A]
Gibt es bei der Installation für Git für MacOS via https://git-scm.com/download/mac ein Best Practice, d.h. ist Homebrew besser als Xcode...?

**Comment by phschaer** [comment]
Homebrew würde ich Ihnen auf dem Mac sowieso uneingeschränkt empfehlen: https://brew.sh/index_de

Danach ist die Installation von Git ein Klacks: `brew install git` - Aber eigentlich nicht nötig, da Git automatisch bei MacOS mitgeliefert wird. Allerdings in einer alten Version. Daher lohnt sich Homebrew so oder so.
